src/settings/Settings.ts

```diff
@@ -847,6 +851,10 @@ export const SETTINGS: {[setting: string]: ISetting} = {
     [UIFeature.ShowSettingAllowFallbackCallAssist]: {
         supportedLevels: LEVELS_UI_FEATURE,
         default: true
+    },
+    [UIFeature.ShowShareAnalyticsInformation]: {
+        supportedLevels: LEVELS_UI_FEATURE,
+        default: false
```

Comment:
Please default to true here, and set to false in communicate-config.json

Reply:
I have removed this as per your other code review suggestion.
Note: I only introduced this to make it more tight in check but by agreeing to minimizing change demands for removal of this. :) .